ext/mixed/js/core.js

```diff
@@ -288,6 +293,42 @@ const yomichan = (() => {
             this.trigger('orphaned', {error});
         }
 
+        getTemporaryListenerResult(eventHandler, userCallback, timeout=null) {
+            if (
```

Comment:
```js
if (!(
  typeof eventHandler.addListener === 'function' &&
  typeof eventHandler.removeListener === 'function'
))
```
is probably more accurate

Reply:
https://github.com/FooSoft/yomichan/pull/417/commits/ff6b79fdc7ef3fd7be4384dec570138e0ad27012

Reply:
Should be && not || now

Reply:
Sorry, that was careless. I tested it and assumed everything was fine because OR short circuited on the first check. https://github.com/FooSoft/yomichan/pull/417/commits/22a97d916fc6ecab1200b0ffea18cf2d5c9923d4